Services/DataX.Utilities.JobRunner/DataX.Utilities.JobRunner/BaseTableEntity.cs

```diff
@@ -0,0 +1,48 @@
+﻿// *********************************************************************

```

Comment:
Isn't this a dup of Services/DataX.Utilities.JobRunner/BaseTableEntity.cs?

Reply:
Updated